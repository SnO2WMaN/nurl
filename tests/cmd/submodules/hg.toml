args = [
    "http://www.octave.org/hg/octave",
    "release-7-1-0",
    "--submodules",
    "--fetcher",
    "fetchhg",
]
