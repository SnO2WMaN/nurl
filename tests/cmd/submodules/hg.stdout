fetchhg {
  url = "http://www.octave.org/hg/octave";
  rev = "release-7-1-0";
  sha256 = "sha256-zNpgpPPGAoIpSIRx39wJpEyLbxjJBsnFjUy47hGiZLs=";
  fetchSubrepos = true;
}